### DISCUSSION TITLE
hide source type from popup info

### DISCUSSION THREAD

**Discussion by wjj1928** [minad/corfu#523] [Q&A]
Current popup has the source field (e.g.  Dict,, Dabbrev strings) at the end, but I would like to hide/remove it.
Any option available to hide it?

![image](https://github.com/user-attachments/assets/c1771c06-cabd-46b3-9d72-6cc25acbb198)

**Comment by jdtsmith** [comment]
You can alter the CAPF you are using to remove the annotation function, with cape.
